bus-arrival/index.html

```diff
@@ -1,7 +1,10 @@
 <!DOCTYPE html>
 <meta charset="utf-8">
 <meta name="viewport" content="width=device-width, initial-scale=1.0">
+<meta name="apple-mobile-web-app-capable" content="yes">
+<meta name="apple-mobile-web-app-title" content="Bus arrival times">
 <title>Bus arrival times</title>
+<link rel="apple-touch-icon" href="../icons/icon-192.png">
```

Comment:
Would be great if can move all these meta-related tags below everything else since they are less important than the `<link>` tags. The root `index.html` is an example of how I order them.